### ISSUE TITLE
Teamseiten Ansicht

### ISSUE THREAD

**Issue by DrCha0z** [Contentify/Contentify#490]
Hey, 
in der Teamseiten ánsicht hat man ja die möglichkeit einen Text über das backend anzeigen zu lassen. Dieser ist aber nicht formatiert. Also ich kann keine links anzeigen bzw schreibt er ja auch alle hintereinander. Gibt es da schon eine änderung? 

Hier mal ein Bild wie es ja aktuell ausehen würde

https://puu.sh/CKckh/0006bf0780.jpg

**Comment by chriskonnertz** [comment]
Hallo,

versuche ich mir heute Abend anzuschauen.

**Comment by DrCha0z** [comment]
Hey, auch dieses Thema ist ja auch erledigt mit den Files, die ich ja schon im Thema mit den Team bildern hoch geladen habe. Fals du es nicht gesehen haben solltest.

**Comment by chriskonnertz** [comment]
Hey ging es da um die Beschreibung von Membern? Die dann auf der Team-Seite angezeigt wird?

**Comment by DrCha0z** [comment]
Nein, es ging um das Textfeld über den Membern, bei mir habe ich dort inzwischen drei Spalten für texte mit verlinkung drinnen. 

Ich habe es al in den Screenshots rot makiert. 

http://www.purfuerst.rocks/FTP/Bilder/1.png
http://www.purfuerst.rocks/FTP/Bilder/2.png

Die geänderten files dürfen auch in der zip sein von der match/teamansicht.
